web/js/codeworld.js

```diff
@@ -200,8 +202,15 @@ function initCodeworld() {
 
                         if (request.status == 400 || request.status ==
                             200) {
-                            callback(parseCompileErrors(request
-                                .responseText))
+                            let errors = parseCompileErrors(request.responseText);
+                            // Runtime errors is priority source compared to
```

Comment:
Runtime errors only matter if the current source generation is the same as the running source generation.  The running source generation is already tracked, so this is just a matter of doing a comparison.

Reply:
Thanks for explanation.